coverage/lcov-report/client/src/components/Overview/Cart/Cart.jsx.html

```diff
@@ -0,0 +1,124 @@
+
+<!doctype html>
```

Comment:
You might want to unstage the coverage directory in git.

Reply:
Good call, I did that git process where we remove all staged files and then re-add them, which should fix the ignored files.